doc/input-output-reference/src/overview/group-hybrid-model.tex

```diff
@@ -0,0 +1,43 @@
+\section{Group - Hybrid Model}\label{group---hybrid-model}
+
+This object is to provide a hybrid model feature to enhance the accuracy of the building energy simulation for existing buildings. The hybrid model feature replaces internal thermal mass and infiltration air flow rates with zone air temperature data. The hybrid model feature calculates internal thermal mass and infiltration air change per hour (ACH) using based on the inverse modeling of the heat balance algorithms. 
+
```

Comment:
Updated the sentence:

The hybrid model feature calculates a temperature capacitance multiplier, equivalent to the zone internal thermal mass and infiltration air change per hour (ACH) based on the inverse modeling of the heat balance algorithms.